spec/jobs/checksum_report_job_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ChecksumReportJob, type: :job do
+  let(:pids) { ['druid:123', 'druid:456'] }
+  let(:groups) { [] }
+  let(:user) { instance_double(User, to_s: 'jcoyne85') }
+  let(:output_directory) { 'tmp/checksum_report_job_success' }
+  let(:output_directory_fail) { 'tmp/checksum_report_job_fail' }
+  # different output_directory so our 'fail' test doesn't inadvertently fail due to the CSV already existing from the 'success' test
+  let(:bulk_action) do
+    create(
+      :bulk_action,
+      action_type: 'ChecksumReportJob',
+      log_name: 'tmp/checksum_report_job_log.txt'
+    )
+  end
+  let(:csv_response) { "druid:123,checksum1,checksum2\ndruid:456,checksum3,checksum4\n" }
+  let(:log_buffer) { StringIO.new }
+  let(:my_conn) { instance_double(Faraday::Connection) }
+
+  before do
+    allow(subject).to receive(:bulk_action).and_return(bulk_action)
+    allow(Ability).to receive(:new).and_return(ability)
+    allow(Faraday).to receive(:new).and_return(my_conn)
+  end
+
+  describe '#perform_now' do
+    context 'with authorization' do
+      let(:ability) { instance_double(Ability, can?: true) }
+
+      it 'calls the presevation_catalog API, writes a CSV file, and records success counts' do
+        response = instance_double(Faraday::Response, success?: true, body: csv_response)
+        allow(my_conn).to receive(:get).and_return(response)
+        subject.perform(bulk_action.id,
+                        output_directory: output_directory,
+                        pids: pids,
+                        groups: groups,
+                        user: user)
+        expect(my_conn).to have_received(:get).with('/objects/checksums', druids: pids, format: 'csv')
+        expect(File.exist?(File.join(output_directory, Settings.CHECKSUM_REPORT_JOB.CSV_FILENAME))).to be_truthy
```

Comment:
This should be `expect(File).to exist(...)`  See https://www.rubydoc.info/gems/rspec-expectations/RSpec/Matchers%3Aexist

Reply:
And if you make this change you can remove the new rspec predicate matcher exclusions from `.rubocop_todo.yml` :point_up: